flow/operations.py

```diff
@@ -101,12 +101,13 @@ def hello_cmd(job):
         )
 
     @wraps(func)
-    def decorated(*jobs):
-        with jobs[0] as job:
+    def decorated(__flow_internal_with_job_argument):
```

Comment:
this was done because when `cmd` is used with `with_job`, the formatting logic assumes that user specified `*jobs` variable argument, but that's us not the user :)

Reply:
I don't understand this change. If you call `func(*jobs)` with a single argument like `func(job)`, it will get `jobs = [job]` as an argument.